### PR TITLE
Change colours of pie chart

### PR DESCRIPTION
**Pull request by DrMichael**
As asked for in https://github.com/adriankumpf/teslamate/issues/608

### CODE REVIEW

File: grafana/dashboards/trip.json [hunk 1]
```diff
@@ -46,7 +113,7 @@
   "panels": [
     {
       "collapsed": false,
-      "datasource": "TeslaMate",
+      "datasource": "${DS_TESLAMATE}",
```
Comment (adriankumpf) [inline grafana/dashboards/trip.json:116]: The `datasource` must not be changed. It would break other setups.

File: grafana/dashboards/trip.json [hunk 2]
```diff
@@ -2806,6 +2740,6 @@
   },
   "timezone": "",
   "title": "Trip",
-  "uid": "FkUpJpQZk",
-  "version": 1
-}
+  "uid": "T_dr9MFMk",
```
Comment (adriankumpf) [inline grafana/dashboards/trip.json:2743]: Please revert this chagne to the `uid`.l

File: grafana/dashboards/trip.json [hunk 3]
```diff
@@ -1,4 +1,70 @@
 {
+  "__inputs": [
```
Comment (adriankumpf) [inline grafana/dashboards/trip.json:2]: I'm not sure where these changes are coming from. It would be nice if you could remove them.

**Review by adriankumpf — CHANGES_REQUESTED** [review]

File: grafana/dashboards/trip.json (hunk 3 above)
Comment (DrMichael) [reply grafana/dashboards/trip.json:2]: Aaaah, I think those strange changes came from switching this on
image

I will make another PR...

**Comment by DrMichael** [comment]
I made a new PR https://github.com/adriankumpf/teslamate/pull/970. I tried to make minimal changes...

**Pull request closed without merging**